### PR TITLE
[Fastlane.swift] Fix wrong data type when convert Hash-type options to Swift

### PR DESCRIPTION
**Pull request by thii**
Options with `Hash` type were converted to `String` type in Swift.

### CODE REVIEW

File: fastlane/lib/fastlane/swift_fastlane_function.rb [hunk 1]
```diff
@@ -106,6 +106,8 @@ def get_type(param: nil, default_value: nil, optional: nil, param_type_override:
         type = "Bool"
       elsif default_value.kind_of?(Array)
         type = "[String]"
+      elsif default_value.kind_of?(Hash)
```
Comment (taquitos) [inline fastlane/lib/fastlane/swift_fastlane_function.rb]: Great catch

**Review by taquitos — CHANGES_REQUESTED** [review]
This is great, thank you. 
If you can remove the changes to the `fastlane/swift/Fastlane.swift` file, we can get this merged.
Most of those `.swift` files are automatically generated during deployment time 🎉

**Review by taquitos — APPROVED** [review]

**Comment by taquitos** [comment]
🎈:goat:

**Comment by antonino-u** [comment]
@taquitos curious: this has apparently made it into 2.80.0 but the autogenerated files haven't been updated. There are also other fixes which I'm sure have been made that aren't in Fastlane.swift. Is there a way for me to force the project to regenerate them? Does someone have to do something to increment FastlaneRunnerAPIVersion?

**Comment by taquitos** [comment]
You're right, it should have made it into this: https://github.com/fastlane/fastlane/pull/11760/files

If you check out the fastlane repo, you can run `fastlane generate_swift_api` and that will generate the files. It works for me, so I wonder if our deploy script doesn't work right.

**Comment by taquitos** [comment]
Ahh, figured it out, we merged this PR after we version bumped. The version bump does the fastlane swift generation.

**Pull request merged**